pkgs/servers/clickhouse/default.nix

```diff
@@ -6,6 +6,7 @@
 , ninja
 , python3
 , perl
+, nasm
 , yasm
 , nixosTests
 , darwin
```

Comment:
This is supposed to go to L15, but GH doesn't let me: `rustSupport` doesn't seem to be for blake3 only anymore, and we should check if it still builds at all without it?

Reply:
I updated the comment, and verified it still builds without it.